md/cfcli/apps.md

```diff
@@ -0,0 +1,33 @@
+# Organizations and Spaces
+
+The applications and users of a Cloud Foundry platform are split into Organizations and Spaces. This gives a way to built a multi-tenant environment with minimal headache since each of these organizations and spaces can be given it's own resource quotas and access controls.
+
+As part of your free trial, you have administrator rights for your own organization with a few different spaces by default. By default, there are a few applications running in the "sample" space (selected at login).
```

Comment:
Do we maybe rather want this material to be independent of the exploration environment, i.e. no explicit references to our specific target environment?

Reply:
I think we don't? There is other material that we can link to for more general usage. I do think we need to note what is "how we set it up" vs "cloud foundry in general" when there's discrepancy.

Reply:
Added action item to trello to track this open question